### PR TITLE
Loading screen update

### PR DESCRIPTION
**Pull request by qiandrewj**
# Summary

This PR works on the Loading feature.

## PR Type

- [X] 🍕 Feature
- [ ] 🐛 Bug Fix
- [] 📝 Documentation Update
- [X] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile + Desktop Screenshots & Recordings
IMG_4539
## QA - Test Plan
Checked to make sure that loading screen is updated with new wheel in all screens: Course screen, Home screen, Admin screen
## Breaking Changes & Notes
Loading wheel graphics can be updated by changing CSS files with new design choices
## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 notion
- [ ] 🍕 ...
- [ ] 📕 ...
- [X] 🙅 no documentation needed

## What GIF represents this PR?

[gif](https://www.icegif.com/wp-content/uploads/tom-and-jerry-icegif-4.gif)

### CODE REVIEW

File: client/src/modules/Admin/Components/Admin.tsx [hunk 1]
```diff
@@ -450,7 +451,7 @@ export const Admin = () => {
 
   function adminLogin() {
     if (loading) {
-      return <div>Loading...</div>
+      return <div><Loading /></div>
```
Comment (wizhaaa) [inline client/src/modules/Admin/Components/Admin.tsx:454]: I think you can just directly return one the Loading component, not being wrapped in div tags for cleanliness.

File: client/src/modules/Results/Components/ResultsDisplay.jsx [hunk 2]
```diff
@@ -323,7 +324,7 @@ export default class ResultsDisplay extends Component {
       <div className={styles.container}>
         <h1> Search Results </h1>
         {/* Case where results are still being loaded */}
-        {this.props.loading === true && <div> Loading ... </div>}
+        {this.props.loading === true && <div><Loading /></div>}
```
Comment (wizhaaa) [inline client/src/modules/Results/Components/ResultsDisplay.jsx:327]: Same here~

**Review by wizhaaa — APPROVED** [review]
Awesome work! Very simple and relatively straightforward task with not too much impact  - but nonetheless still important for a well-polished site is a good loading state. The ideal is to never see it and rarely see it - but for the rare times we do - either slow internet, long calculations, or in any edge case we can't think of, we want a user to know that they are waiting for something. 

Love how pretty clean and simple the implementation is too!

**Review by leihelen — COMMENTED** [review]
I think the loading screen works very well and it is implemented very well with clean, simple code. Overall it looks great and having this implemented will really improve user experience! One suggestion I have is just to maybe change the name spinner to be more specific, for example something like "loading-spinner" or "loading-icon" might provide more clarity as to what it is for.

**Review by leihelen — APPROVED** [review]
I think the loading screen works very well and it is implemented very well with clean, simple code. Overall it looks great and having this implemented will really improve user experience! One suggestion I have is just to maybe change the name spinner to be more specific, for example something like "loading-spinner" or "loading-icon" might provide more clarity as to what it is for.

**Pull request merged**